tests/std/tests/P2231R1_complete_constexpr_optional_variant/test.cpp

```diff
@@ -126,16 +126,27 @@ constexpr bool test_optional() {
     }
 
     { // construction from optional with convertible types
-        optional<T> constructed{42};
         optional<int> input{42};
 
-        constructed = input;
-        assert(constructed.has_value());
-        assert(*constructed == 42);
+        optional<T> copy_constructed{input};

```

Comment:
The names `copy_constructed`, `move_constructed`, `copy_assigned`, and `move_assigned` are misleading, given that none of the operations performed on these objects is the operation named. Could we rename these?

Reply:
Renamed globally